splunk-otel-android/src/main/java/com/splunk/rum/SessionIdChangeTracer.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.rum;
+
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.trace.Tracer;
+
+final class SessionIdChangeTracer implements SessionIdChangeListener {
+
+    static final AttributeKey<String> PREVIOUS_SESSION_ID_KEY = stringKey("splunk.previous.rumSessionId");
```

Comment:
Ideally I'd like to name it `splunk.rum.previous_session_id`, but it wouldn't match `splunk.rumSessionId` (which _ideally_ should be named `splunk.rum.session_id`...) so I just put together something like this. I'm not really convinced this is a good name though.

Reply:
Since Android is the first to implement this feature, I think we can apply some leverage to the backend team to use what we like, and ask iOS/web to conform if they agree.